src/components/sections/students/list-table/index.tsx

```diff
@@ -12,10 +13,12 @@ import { columns } from './columns';
 function StudentsListTable() {
   const pagination = usePagination();
   const filters = useTableFilters();
+  const status = useTableStatus();
```

Comment:
I think it's better to just use the pre-existing `useTableFilters` hook to add the license file status filter and any other filters in the future.
```suggestion
  const filters = useTableFilters();
```